docs/en/observability/logs-threshold-alert.asciidoc

```diff
@@ -14,8 +14,8 @@ image::images/log-threshold-alert.png[Log threshold alert configuration]
 The comparators available for conditions depend on the chosen field. The combinations available are:
 
 - Numeric fields: *more than*, *more than or equals*, *less than*, or *less than or equals*.
-- Aggregatable fields: *is* and *is not*.
-- Non-aggregatable fields: *matches*, *does not match*, *matches phrase*, *does not match phrase*.
+- Aggregable fields: *is* and *is not*.
```

Comment:
I'm not sure we should change this. It's used as a field in some response bodies and is used all over the docs:

https://www.elastic.co/guide/en/elasticsearch/reference/8.2/search-field-caps.html#search-field-caps-api-response-body

Reply:
👍  My (brief) research suggested that "aggregable" may be slightly more widely used than "aggregatable", but either is acceptable.